### PR TITLE
Misc. explosives fixes

### PR DESCRIPTION
**Pull request by NoooneyDude**
CL: [Improvement] Explosives UIs now have an exit button.
CL: [Improvement] Explosives UIs can now be moved around.
CL: [Fix] Gets the pizza bomb somewhat functional. It now detonates on opening it, if armed.
CL: [Fix] Fixed a few cases of explosives buttons not working.
CL: [Fix] Fixed a few cases of explosives UI sprites not updating correctly.
CL: [Fix] Fixed a case or two of explosive timer UI oddness.
CL: [Improvement] The powersink now has examination states to assist with usage.
- Adds a chat message or two for player awareness.
- Explosive arming at spawn is now exposed to the inspector.
- Adds a netmessage to disable a component. Practically useless though, needs further work.

part of #6149 

> Tested with a client on headless.

### CODE REVIEW

File: UnityProject/Assets/Scripts/Items/Weapons/AttachableExplosive.cs [hunk 1]
```diff
@@ -76,8 +76,23 @@ private bool CanAttatchToTarget(Matrix matrix, RegisterTile tile)
 
 		public bool WillInteract(PositionalHandApply interaction, NetworkSide side)
 		{
-			if (DefaultWillInteract.Default(interaction, side) == false
-			    || isArmed == true || pickupable.ItemSlot == null || interaction.Intent != Intent.Harm) return false;
+			if (DefaultWillInteract.Default(interaction, side) == false || pickupable.ItemSlot == null) return false;
+
+			// Why do we prevent mounting if it is armed?
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/Items/Weapons/AttachableExplosive.cs:81]: @MaxIsJoe

**Comment by corp-0** [comment]
Looks good. I'm waiting for Max to answer those whys you left there

File: UnityProject/Assets/Scripts/Items/Weapons/AttachableExplosive.cs (hunk 1 above)
Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/Weapons/AttachableExplosive.cs:81]: ayo dont remove the harm intent, its intentional design

Comment (MaxIsJoe) [reply UnityProject/Assets/Scripts/Items/Weapons/AttachableExplosive.cs:81]: actually nevermind apparently he added a message to tell the player about it, thats nice.

**Comment by MaxIsJoe** [comment]
Seems fine so far, nothing that catched my eye code wise or gameplay wise when I tested this.

This is good to go, massive thanks Nooney 👍 ❤️

**Review by Bod9001 — APPROVED** [review]

**Pull request merged**